base/src/collection-service.ts

```
@@ -175,21 +175,43 @@ class CollectionService implements Service {
           case '$populate': {
             const fields = Array.isArray(value) ? value : [value]
             const schema = await this.collection.schema
+
             filter.$populate = fields.map((field) => {
+              const err = new BadRequest(`Cannot populate field \`${field}\``)
```

Better to throw an error so that you don't get take by surprise